cmd/fs-v1-metadata.go

```diff
@@ -375,6 +383,8 @@ func checkFormatSanityFSV2(fsPath string) error {
 // Check for sanity of FS format for a given version.
 func checkFormatSanityFS(fsPath string, fsFormatVersion string) (err error) {
 	switch fsFormatVersion {
+	case fsFormatV1:
```

Comment:
So I just updated the patch. Though FSV1's check is a no-op currently, it helps as a reference for future development - this way every version's sanity check can be useful.